documentation-samples/multinic/azuredeploy.json

```diff
@@ -0,0 +1,287 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vnetName": {
+      "type": "string",
+      "defaultValue": "WTestVNet",
+      "metadata": {
+        "description": "Name for the new VNet."
+      }
+    },
+    "backEndSubnetName": {
+      "type": "string",
+      "defaultValue": "BackEnd",
+      "metadata": {
+        "description": "Name for the back end subnet."
+      }
+    },
+    "backEndNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-BackEnd",
+      "metadata": {
+        "description": "Name for the NSG used to allow DB access, and block Internet."
+      }
+    },
+    "remoteAccessNSGName": {
+      "type": "string",
+      "defaultValue": "NSG-RemoteAccess",
+      "metadata": {
+        "description": "Name for the NSG used to allow remote access."
+      }
+    },
+    "stdStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstoragestd",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstorageprm",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Premium_GRS",
+        "Premium_ZRS"
+      ]
+    },
+    "osType": {
+      "type": "int",
+      "defaultValue": 0,
+      "allowedValues": [
+        0,
+        1
+      ],
+      "metadata": {
+        "description": "Type of OS to use for VMs: 0 - Windows; 1 - Ubuntu."
+      }
+    },
+    "dbCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of database VMs to be deployed to the backend end subnet."
+      }
+    },
+    "parentRG": {
+      "type": "string",
+      "defaultValue": "IaaSStory",
+      "metadata": {
+        "description": "Name of resource group containing vnet and front end resources."
+      }
+    }
+  },
+  "variables": {
+    "dbVMSettings": [
+      {
+        "vmSize": "Standard_DS3",
+        "adminUsername": "adminuser",
+        "adminPassword": "adminP@ssw0rd",
```

Comment:
As a security best practice, please let the user choose username and password instead of hardcoding these in a public template